electron/js/menu/system.js

```diff
@@ -267,6 +267,18 @@ var win32Template = {
   ],
 };
 
+var linuxTemplate = {
+  label: config.NAME,
```

Comment:
Oh no! I just pushed changes to make the menu 'File' (though you might need to double-check my German!). Anyway, on Monday let me know what the designers prefer and I can revert if necessary.